tests/csm/rest/test_capacity_quota.py

```diff
@@ -641,3 +646,354 @@ def test_40640(self):
+        self.log.info("##### Test ended -  %s #####", test_case_name)
+
+    @pytest.mark.skip("Feature not ready")
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-41757')
+    def test_41757(self):
+        """
+        Test GET capacity stats for delete IAM , Purge False user using admin users
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("Pre-condition: Create multiple objects (N) on 1 bucket")
+        test_cfg = self.csm_conf["test_41757"]
+        max_size = test_cfg["max_size"]
+        random_size = self.cryptogen.randrange(1, max_size)
+        num_objects = math.floor(max_size / random_size)
+        for _ in range(0, num_objects):
+            resp = s3_misc.create_put_objects(self.obj_name, self.bucket,
+                                              self.akey, self.skey, object_size=random_size)
+            assert resp, "Put object Failed"
+        self.log.info("Step 1: Delete iam user")
+        self.log.info("Verify Delete IAM user: %s with access key: %s and secret key: %s",
+                      self.user_id, self.akey, self.skey)
+        purge_data = False
+        response = self.csm_obj.delete_iam_user_rgw(self.user_id, purge_data)
+        assert response.status_code == HTTPStatus.OK, \
+                           "Status code check failed for user deletion"
+        self.log.info("Step 2: Perform GET API to get capacity usage stats")
+        resp = self.csm_obj.get_capacity_usage("user", self.user_id)
+        assert resp.status_code == HTTPStatus.BAD_REQUEST, \
+                           "Status code check failed for user deletion"
+        #TODO: Error code and message check part
+        self.log.info("##### Test ended -  %s #####", test_case_name)
+
+    @pytest.mark.skip("Feature not ready")
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-41758')
+    def test_41758(self):
+        """
+        Test GET capacity stats for delete IAM , Purge True user using admin users
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("Pre-condition: Create multiple objects (N) on 1 bucket")
+        test_cfg = self.csm_conf["test_41158"]
+        max_size = test_cfg["max_size"]
+        random_size = self.cryptogen.randrange(1, max_size)
+        num_objects = math.floor(max_size / random_size)
+        for _ in range(0, num_objects):
+            resp = s3_misc.create_put_objects(self.obj_name, self.bucket,
+                                              self.akey, self.skey, object_size=random_size)
+            assert resp, "Put object Failed"
+        self.log.info("Step 1: Delete iam user")
```

Comment:
Put object is the step 1. Can u correct the step nunmber

Reply:
done